lib/mongo-stream.test.js

```diff
@@ -60,8 +60,7 @@ describe(__filename, function () {
         name: 'logger',
         msg: 'test'
       }), null, function (err) {
-        expect(err).to.be.an('Error');
-        expect(err.toString()).to.contain('fake mongodb error');
+        expect(err).to.not.exist;
```

Comment:
Minor, but I started doing `if (err) { return done(err); }` in my tests, just so mocha prints out the entire error/stack instead of an abbreviated version.